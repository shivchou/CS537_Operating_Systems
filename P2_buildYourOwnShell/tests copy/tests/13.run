../solution/wsh tests/13.wsh
