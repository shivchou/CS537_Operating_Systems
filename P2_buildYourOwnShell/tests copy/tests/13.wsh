echo hello world | tr 'a-z' 'A-Z'
echo hello world | tr 'a-z' 'A-Z' | sed 's/HELLO/HI/'
echo foo | notacommand | tr 'a-z' 'A-Z'
echo test | echo again
echo hi | | tr 'a-z' 'A-Z'
echo a | tr 'a-z' 'A-Z' | tr 'A-Z' 'a-z'
