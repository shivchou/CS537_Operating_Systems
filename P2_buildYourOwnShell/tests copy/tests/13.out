HELLO WORLD
HI WORLD
again
a
